projects/angular2-draggable/src/lib/angular-resizable.directive.ts

```diff
@@ -378,7 +378,8 @@ export class AngularResizableDirective implements OnInit, OnChanges, OnDestroy,
       position: {
         top: this._currPos.y,
         left: this._currPos.x
-      }
+      },
+      direction: this._direction
```

Comment:
Prefer returning a deep clone of `this._direction`

Reply:
Want me to push a change? I noticed you merged it

Reply:
No, thanks. I've made the change.